truck_signs_designs/settings/base.py

```diff
@@ -23,8 +25,8 @@
 
 
 
-
-ALLOWED_HOSTS = ['0.0.0.0', 'localhost', '127.0.0.1',]
+load_dotenv()
+ALLOWED_HOSTS = ['0.0.0.0', 'localhost', '127.0.0.1', os.getenv("IP_ADDR")]
```

Comment:
@jonathan-priebe we do not want to allow localhost in Production.

```suggestion
ALLOWED_HOSTS = [os.getenv("IP_ADDR")]
```

Reply:
I have also added a split feature to include multiple IP addresses.